sim/play.py

```diff
@@ -78,49 +82,85 @@ def play(args: argparse.Namespace) -> None:
         export_policy_as_jit(ppo_runner.alg.actor_critic, path)
         print("Exported policy as jit script to: ", path)
 
-    # export policy as a onnx module (used to run it on web)
-    if args.export_onnx:
-        path = ppo_runner.alg.actor_critic
-        convert_model_to_onnx(path, ActorCfg(), save_path="policy.onnx")
-        print("Exported policy as onnx to: ", path)
+    # # export policy as a onnx module (used to run it on web)
```

Comment:
Why commented?

Reply:
I think its outdated / something that was changed/handled in kinfer, when I run it I get this. Maybe wesley knows more, but low priority for now imo.

```
  loaded_dict = torch.load(path)
Exported policy as jit script to:  .
Traceback (most recent call last):
  File "/opt/anaconda3/envs/ksl/lib/python3.8/site-packages/torch/serialization.py", line 564, in _check_seekable
    f.seek(f.tell())
  File "/opt/anaconda3/envs/ksl/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1729, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'ActorCritic' object has no attribute 'seek'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "sim/play.py", line 332, in <module>
    play(args)
  File "sim/play.py", line 94, in play
    convert_model_to_onnx(path, ActorCfg(), save_path="policy.onnx")
  File "/home/kasm-user/ali_repos/sim/sim/model_export.py", line 220, in convert_model_to_onnx
    all_weights = torch.load(model_path, map_location="cpu", weights_only=True)
  File "/opt/anaconda3/envs/ksl/lib/python3.8/site-packages/torch/serialization.py", line 1065, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/opt/anaconda3/envs/ksl/lib/python3.8/site-packages/torch/serialization.py", line 473, in _open_file_like
    return _open_buffer_reader(name_or_buffer)
  File "/opt/anaconda3/envs/ksl/lib/python3.8/site-packages/torch/serialization.py", line 458, in __init__
    _check_seekable(buffer)
  File "/opt/anaconda3/envs/ksl/lib/python3.8/site-packages/torch/serialization.py", line 567, in _check_seekable
    raise_err_msg(["seek", "tell"], e)
  File "/opt/anaconda3/envs/ksl/lib/python3.8/site-packages/torch/serialization.py", line 560, in raise_err_msg
    raise type(e)(msg)
AttributeError: 'ActorCritic' object has no attribute 'seek'. You can only torch.load from a file that is seekable. Please pre-load the data into a buffer like io.BytesIO and try to load from it instead.
```